lib/util/storage.js

```diff
@@ -0,0 +1,34 @@
+// Prefix to use with local storage keys.
+const STORAGE_PREFIX = 'otp'
```

Comment:
This is all very specific for use with localStorage. Would it be advantageous to rewrite this so that we can use a generic persistence worker that can automatically figure out how to read and save data?

Reply:
I think that's the plan in the long run. If you have any specific thoughts on a refactor for right now, I'm open to them.

Reply:
I was thinking that there could be a class with generic methods that could be implemented to initialize a new session and then get and save data. The config could specify the type of persistence manager and any need keys, etc.

Reply:
@evansiroky, what do you think about just adding a `TODO` here about this? My concern is that to truly make something like you're describing it might be worthwhile to have some other persistence strategy to build against. I think given that our immediate task is to store data in local storage, we should aim to build that and encapsulate it as much as possible for the next project, when we expect to swap in a different persistence strategy. There are factors with future projects that we don't have complete information about, so I'm worried we would waste our time writing too much code for hypothetical systems.

Reply:
Let's create an issue for OTP-RR to be able to save user settings to something other than localstorage and call it good for now.